web-client/src/views/CaseDetailEdit/ReviewSavedPetition.jsx

```diff
@@ -58,8 +58,11 @@ export const ReviewSavedPetition = connect(
             </h1>
           </Focus>
 
-          {reviewSavedPetitionHelper.hasOrders && (
-            <OrdersNeededSummary caseInformation={form} />
+          {(reviewSavedPetitionHelper.ordersAndNoticesNeeded.length > 0 ||
```

Comment:
this logic should be moved to a computed

Reply:
Refactor completed.